libs/providers/ofrep-web/package.json

```diff
@@ -13,6 +13,6 @@
   },
   "dependencies": {
     "undici": "^5.0.0",
-    "@openfeature/ofrep-core": "0.2.0"
+    "@openfeature/ofrep-core": "1.0.0"
```

Comment:
Might as well do this now so we can more easily communicate breaking changes. This is an "internal" lib, we can break it all we want.